testgen/generators/segmenter_gen.js

```diff
@@ -0,0 +1,181 @@
+/* Generate segmenter test data with the following dimensions:
+
+   1. locale
+   2. type (conjunction, disjunction, unit)
+   3. style (long, short, narrow)
+*/
+
+// Set up Node version to generate data specific to ICU/CLDR version
+// e.g., `nvm install 21.6.0;nvm use 21.6.0` (ICU 74)
+
+const common_fns = require("./common.js");
+const gen_hash = require("./generate_test_hash.js");
+
+const fs = require('node:fs');
+
+let debug = false;
+
+const test_type = "segmenter";
+
+const granularity = ['grapheme', 'word', 'sentence'];
```

Comment:
add comment that Intl.JS doesn't support line breaking. include this link for reference to the reason:
https://github.com/tc39/proposal-intl-segmenter#why-is-line-breaking-not-included

Reply:
No, I'll include line break as an option, then mark as unsupported in the NodeJS executor.